libs/dh/metering-point/feature-tabs/src/lib/processes-tab-content/dh-processes-table/dh-processes-table.component.html

```diff
@@ -0,0 +1,150 @@
+      </mat-cell>
+    </ng-container>
+
+    <!-- Name Definition -->
+    <ng-container matColumnDef="name">
+      <mat-header-cell
+        *matHeaderCellDef
+        mat-sort-header
+        [sortActionDescription]="
+          'meteringPoint.overview.processesTab.sortDescription.name' | transloco
+        "
+        class="watt-label"
+      >
+        {{ 'meteringPoint.overview.processesTab.headers.name' | transloco }}
+      </mat-header-cell>
+      <mat-cell *matCellDef="let row" data-testId="processName">
+        <div class="watt-text-s">
+          {{ row.name }}
+        </div>
+      </mat-cell>
+    </ng-container>
+
+    <!-- Created Date Definition -->
+    <ng-container matColumnDef="createdDate">
+      <mat-header-cell
+        *matHeaderCellDef
+        mat-sort-header
+        [sortActionDescription]="
+          'meteringPoint.overview.processesTab.sortDescription.createdDate'
+            | transloco
+        "
+        class="watt-label"
+      >
+        {{
+          'meteringPoint.overview.processesTab.headers.createdDate' | transloco
+        }}
+      </mat-header-cell>
+      <mat-cell *matCellDef="let row">
+        <watt-icon
+          name="date"
+          [size]="iconSize.Small"
+          class="date-icon"
+        ></watt-icon>
+        <span class="watt-text-s">{{ row.createdDate | dhDateTime }} </span>
+      </mat-cell>
+    </ng-container>
+
+    <!-- Effective Date Definition -->
+    <ng-container matColumnDef="effectiveDate">
+      <mat-header-cell
+        *matHeaderCellDef
+        mat-sort-header
+        [sortActionDescription]="
+          'meteringPoint.overview.processesTab.sortDescription.effectiveDate'
+            | transloco
+        "
+        class="watt-label"
+      >
+        {{
+          'meteringPoint.overview.processesTab.headers.effectiveDate'
+            | transloco
+        }}
+      </mat-header-cell>
+      <mat-cell *matCellDef="let row">
+        <watt-icon
+          name="date"
+          [size]="iconSize.Small"
+          class="date-icon"
+        ></watt-icon>
+        <span class="watt-text-s">{{ row.effectiveDate | dhDate }} </span>
+      </mat-cell>
+    </ng-container>
+
+    <!-- Status Definition -->
+    <ng-container matColumnDef="status">
+      <mat-header-cell
+        *matHeaderCellDef
+        mat-sort-header
+        [sortActionDescription]="
+          'meteringPoint.overview.processesTab.sortDescription.status'
+            | transloco
+        "
+        class="watt-label"
+      >
+        {{ 'meteringPoint.overview.processesTab.headers.status' | transloco }}
+      </mat-header-cell>
+      <mat-cell *matCellDef="let row">
+        <div class="watt-text-s">
+          {{ row.status }}
+        </div>
+      </mat-cell>
+    </ng-container>
+
+    <!-- Has Errors Definition -->
+    <ng-container matColumnDef="hasErrors">
+      <mat-header-cell *matHeaderCellDef class="watt-label"> </mat-header-cell>
```

Comment:
question: Is it necessary to add an empty `mat-header-cell` element?

Reply:
I think a cell definition is required. Not sure if there's a shorter version to do so.

Reply:
My mistake. It is necessary. Well, unless we're going to put some text inside the header cell, then we can get rid of the `class`.